### PR TITLE
adding hxnfly, hxntools, ppmac recipes

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: 
https://github.com/nsls-ii-forge/hxnfly-feedstock
https://github.com/nsls-ii-forge/ppmac-feedstock
https://github.com/nsls-ii-forge/hxntools-feedstock
https://github.com/nsls-ii-forge/nslsii-feedstock
https://github.com/nsls-ii-forge/pyolog-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin 

pyolog is failing on windows, but it's a noarch package. not sure where to add the # [not win] line since it does not have a command section like the previous packages we've done.

File: recipes/hxnfly/LICENSE [hunk 1]
```diff
@@ -0,0 +1,1420 @@
...
+<!DOCTYPE html>
+<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark">
+  <head>
+    <meta charset="utf-8">
+  <link rel="dns-prefetch" href="https://github.githubassets.com">
+  <link rel="dns-prefetch" href="https://avatars.githubusercontent.com">
+  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
+  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">
+
+
```
Comment (mrakitin) [inline recipes/hxnfly/LICENSE:16]: This file does not look good. Please replace it with the raw file from https://raw.githubusercontent.com/nsls-ii-forge/hxnfly-feedstock/master/recipe/LICENSE.

File: recipes/ppmac/LICENSE [hunk 2]
```diff
@@ -0,0 +1,1357 @@
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark">
+  <head>
```
Comment (mrakitin) [inline recipes/ppmac/LICENSE:9]: Same problem with this LICENSE file.

File: recipes/pyolog/LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,1295 @@
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark">
+  <head>
```
Comment (mrakitin) [inline recipes/pyolog/LICENSE.md:9]: Same issue with the license file here.

**Review by mrakitin — CHANGES_REQUESTED** [review]
@kbeeperez, as we earlier saw in one of the PRs, these entry points should be explicitly added to the recipe:
https://github.com/NSLS-II/pyOlog/blob/a85d32753fd30e7beca843a3f24071e79968d604/setup.py#L19-L22.

Please see my concerns about license files.

**Comment by mrakitin** [comment]
@conda-forge/staged-recipes, once the aforementioned issues are resolved, I agree to be a maintainer of all relevant packages in this PR.

**Review by mrakitin — APPROVED** [review]

**Comment by mrakitin** [comment]
@kbeeperez, the entry points for pyolog should be fixed before it goes in. Thanks!

**Comment by kbeeperez** [comment]
@mrakitin Ok, do I need to do any formatting for the entry points? It goes under the `number: 0` correct?

**Comment by mrakitin** [comment]
The format should be similar to https://github.com/conda-forge/staged-recipes/pull/15516/files#diff-64119240494e3d6465bfdd65c22f8c48663fe4acf330b01c31befd59be32ae47R15-R17.

Looks like conda only accepts the `console_scripts` implicitly, and I don't know how to specify the `gui_scripts` there. Probably we can include everything under one implicit type of `console_scripts`, but it'd be great to hear the options from @dopplershift or other CF folks.

**Comment by mrakitin** [comment]
That's what `grayskull pypi pyolog` generates:

```yaml
build:
  number: 0
  noarch: python
  entry_points:
    - olog = pyOlog.cli:main
    - ologgui = pyOlog.gui:main
  script: {{ PYTHON }} -m pip install . -vv
```
I think we can just go with those entry points 🙂.

**Comment by mrakitin** [comment]
@kbeeperez, I think the `pyolog` recipe can be made noarch, and the test import replaced with the test command as I demonstrated in https://github.com/nsls-ii-forge/pyolog-feedstock/pull/3:
```yaml
test:
  commands:
    - python -c "import os; os.environ['USER'] = os.getenv('USER', os.getenv('USERNAME', 'test')); print(os.environ['USER']); import pyOlog"
```

The problem is that [this line](https://github.com/NSLS-II/pyOlog/blob/a85d32753fd30e7beca843a3f24071e79968d604/pyOlog/conf.py#L25) of the pyOlog code does not properly take into account the user name variable on Windows (`%USERNAME%` vs. `$USER` on Linux/Mac). I'll submit a PR to fix it in the package, but meanwhile we can use this temporary workaround.

**Comment by mrakitin** [comment]
@conda-forge/staged-recipes, would you rather prefer us to create a patch for the `pyolog` issue on Windows? I feel it'll be more sustainable before the next version of pyOlog is released with the correct fix.

**Comment by mrakitin** [comment]
I went ahead and created a PR in the project to fix the username handling on Windows (see NSLS-II/pyolog#18). The patch was tested in https://github.com/nsls-ii-forge/pyolog-feedstock/pull/3, and here is my PR to this PR branch: https://github.com/kbeeperez/staged-recipes/pull/1.

@kbeeperez, please merge https://github.com/kbeeperez/staged-recipes/pull/1, and we will be able to see the CI result for this update.

File: recipes/nslsii/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - bluesky-kafka
+    - caproto
+    - databroker
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:35]:
```suggestion
build:
  number: 0
  script: {{ PYTHON }} -m pip install . --no-deps -vv

requirements:
  host:
    - python
    - pip
  run:
    - python
    - appdirs
    - bluesky
    - bluesky-kafka  # [not win]
    - caproto
    - databroker
    - ipython
    - ipywidgets
    - matplotlib
    - numpy
    - ophyd
    - psutil
    - pyolog
```

File: recipes/nslsii/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:14]:
```suggestion
```

File: recipes/nslsii/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:20]:
```suggestion
    - python
```

File: recipes/nslsii/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:23]:
```suggestion
    - python
```

File: recipes/nslsii/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    - appdirs
+    - bluesky
+    - bluesky-kafka
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:26]:
```suggestion
    - bluesky-kafka  # [not win]
```

File: recipes/nslsii/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
+  imports:
+    - nslsii
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:39]: @kbeeperez, could you please add more import tests as we did for other recipes?

**Review by mrakitin — COMMENTED** [review]
I suggested to make the `nslsii` package arch-dependent, as it depends on [`bluesky-kafka`](https://github.com/conda-forge/bluesky-kafka-feedstock/blob/4893b279e9b6949c369ffe27ee3eabd2157d4c85/recipe/meta.yaml#L27), that depends on non-Windows [`python-confluent-kafka`](https://github.com/conda-forge/python-confluent-kafka-feedstock/blob/master/recipe/meta.yaml#L17). Let's see how it goes.

Ultimately, we may want to just skip the Windows build completely for `nslsii`, but then we'll have to make the corresponding packages that import it arch-dependent too.

File: recipes/nslsii/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
  skip: true  # [win]
```

**Review by mrakitin — COMMENTED** [review]
The Windows tests fail:
```py
Traceback (most recent call last):
  File "C:\Miniconda\conda-bld\nslsii_1626984835504\test_tmp\run_test.py", line 2, in <module>
    import nslsii
  File "C:\Miniconda\conda-bld\nslsii_1626984835504\_test_env\lib\site-packages\nslsii\__init__.py", line 16, in <module>
    from bluesky_kafka import Publisher
ModuleNotFoundError: No module named 'bluesky_kafka'
Tests failed for nslsii-0.1.3-py36ha15d459_0.tar.bz2 - moving package to C:\Miniconda\conda-bld\broken
WARNING conda_build.build:tests_failed(2956): Tests failed for nslsii-0.1.3-py36ha15d459_0.tar.bz2 - moving package to C:\Miniconda\conda-bld\broken
set PREFIX=C:\Miniconda\conda-bld\nslsii_1626984835504\_test_env
set SRC_DIR=C:\Miniconda\conda-bld\nslsii_1626984835504\test_tmp

%SRC_DIR%>call "%SRC_DIR%\conda_test_env_vars.bat" 

%SRC_DIR%>set "CONDA_SHLVL="   && 

%SRC_DIR%>conda activate "%PREFIX%" 
import: 'nslsii'
TESTS FAILED: nslsii-0.1.3-py36ha15d459_0.tar.bz2
```
I think we need to skip the `nslsii` build for Windows completely.

File: recipes/nslsii/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
+  imports:
+    - nslsii
+    - nslsii.setup
+    - nslsii.versioneer
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:40]: I don't see these files in https://github.com/NSLS-II/nslsii/tree/master/nslsii. Did you go to the `nslsii` dir before running the script to get imports?

Comment (kbeeperez) [reply recipes/nslsii/meta.yaml:40]: Ah, that's my bad I apologize.

**Comment by mrakitin** [comment]
@kbeeperez, I submitted a PR to this PR branch: https://github.com/kbeeperez/staged-recipes/pull/2. That should resolve the issue with the imports on Windows, allowing to still have the `nslsii` package on Windows (that will eliminate the need of making all packages that depend on it arch-dependent).

**Comment by mrakitin** [comment]
The [latest OSX build](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=355286&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=933) failure is unrelated to this PR as it attempted to package `signalslot-0.1.2-on-osx-64` that is in a completely different PR #15690. I guess it's a leftover from the recent merge, not yet cleaned by the bots.

With https://github.com/kbeeperez/staged-recipes/pull/2, I hope to have all 3 platforms' tests to pass.

File: recipes/nslsii/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - appdirs
+    - bluesky
+    - bluesky-kafka  # [not win]
+    - caproto
+    - databroker
+    - ipython
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:29]:
```suggestion
    - h5py
    - ipython
```

File: recipes/ppmac/meta.yaml [hunk 13]
```diff
@@ -0,0 +1,41 @@
+
```
Comment (mrakitin) [inline recipes/ppmac/meta.yaml:1]:
```suggestion
```

File: recipes/pyolog/meta.yaml [hunk 14]
```diff
@@ -0,0 +1,48 @@
+{% set name = "pyOlog" %}
+{% set version = "4.4.3" %}
+{% set sha256 = "b7530236ea8c6ae1f437aea66ef8ce6d2a8d7fca4d1456ecccd29ff842defd0b" %}
```
Comment (mrakitin) [inline recipes/pyolog/meta.yaml:3]:
```suggestion
```

File: recipes/pyolog/meta.yaml [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (mrakitin) [inline recipes/pyolog/meta.yaml:11]:
```suggestion
  sha256: b7530236ea8c6ae1f437aea66ef8ce6d2a8d7fca4d1456ecccd29ff842defd0b
```

File: recipes/nslsii/meta.yaml [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+    - nslsii.common.ipynb.logutils
+    - nslsii.common.ipynb.nbviewer
+    - nslsii.detectors
+    - nslsii.detectors.trigger_mixins
+    - nslsii.detectors.utils
+    - nslsii.detectors.xspress3
+    - nslsii.detectors.zebra
+    - nslsii.devices
+    - nslsii.iocs.eps_two_state_ioc_sim
+    - nslsii.iocs.thermo_sim
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:56]:
```suggestion
    # TODO: uncomment the tests below once https://github.com/NSLS-II/nslsii/pull/125 is merged/released.
    # - nslsii.iocs.eps_two_state_ioc_sim
    # - nslsii.iocs.thermo_sim
```

**Comment by mrakitin** [comment]
🎉

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes This is ready to merge.

**Comment by mrakitin** [comment]
@conda-forge/staged-recipes friends, could this please be merged soon? We plan to test the packages at NSLS-II next week (Wednesday, August 4), it would be great to have the packages from this PR available. Thanks a lot for your cooperation and help!

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes

File: recipes/hxntools/meta.yaml [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+    - h5py
+    - matplotlib
+    - nslsii
+    - numpy
+    - ophyd
+    - pandas
+    - pyepics
+    - scipy
+
+test:
```
Comment (xhochy) [inline recipes/hxntools/meta.yaml:36]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/ppmac/meta.yaml [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.8
+    - paramiko
+
+test:
```
Comment (xhochy) [inline recipes/ppmac/meta.yaml:26]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/nslsii/meta.yaml [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+    - h5py
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
```
Comment (xhochy) [inline recipes/nslsii/meta.yaml:38]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/pyolog/meta.yaml [hunk 20]
```diff
@@ -0,0 +1,47 @@
...
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - ipython
+    - keyring
+    - requests
+    - six
+
+test:
```
Comment (xhochy) [inline recipes/pyolog/meta.yaml:33]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/hxnfly/meta.yaml [hunk 21]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 7c14c5c16ab3c91a0c379f099d065bb09e66291fe81a8676cb66014663cb3717
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/hxnfly/meta.yaml:19]:
```suggestion
    - python >=3.7
```

File: recipes/hxnfly/meta.yaml [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/hxnfly/meta.yaml:22]:
```suggestion
    - python >=3.7
```

File: recipes/hxntools/meta.yaml [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: ff248d65eec4c015d8d0e622ad465523d900ff485cb297ea8e25c23b0a9edb89
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/hxntools/meta.yaml:19]:
```suggestion
    - python >=3.7
```

File: recipes/hxntools/meta.yaml [hunk 24]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/hxntools/meta.yaml:22]:
```suggestion
    - python >=3.7
```

File: recipes/nslsii/meta.yaml [hunk 25]
```diff
@@ -0,0 +1,75 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c409688657771291fb2c32e95045d3f44432402ebfb41dae5bc7188c1d34c075
+  patches:
+    - 0001-import-bluesky_kafka-only-when-needed.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
  skip: true  # [py<37]
```

File: recipes/ppmac/meta.yaml [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+  sha256: 766b8a579b0d56bbe68f0944780ec4a71ddbcdd859c963c565aaeee2d951dd94
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/ppmac/meta.yaml:19]:
```suggestion
    - python >=3.7
```

File: recipes/ppmac/meta.yaml [hunk 27]
```diff
@@ -0,0 +1,44 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/ppmac/meta.yaml:22]:
```suggestion
    - python >=3.7
```

File: recipes/pyolog/meta.yaml [hunk 28]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+  noarch: python
+  entry_points:
+    - olog = pyOlog.cli:main
+    - ologgui = pyOlog.gui:main
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/pyolog/meta.yaml:24]:
```suggestion
    - python >=3.7
```

File: recipes/pyolog/meta.yaml [hunk 29]
```diff
@@ -0,0 +1,51 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (mrakitin) [inline recipes/pyolog/meta.yaml:27]:
```suggestion
    - python >=3.7
```

**Review by mrakitin — CHANGES_REQUESTED** [review]
We observed the failures with `xarray` and Python 3.6 in other places (see e.g. https://github.com/bluesky/databroker/issues/669) that started about a week ago, so the failures in the CI are unrelated to the suggested changes to do `pip check` (that I personally like). So, I am suggesting to change the minimum supported version of Python for all these packages 3.7.

File: recipes/nslsii/meta.yaml [hunk 30]
```diff
@@ -0,0 +1,76 @@
...
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:43]: I suggest removing it as `nslsii 0.1.3 requires bluesky-kafka, which is not installed.`

We skip this dependency here, but not in https://github.com/NSLS-II/nslsii/blob/904e15e77f2174c3d2ff47b7b7984db0bd360bb2/requirements.txt#L3.

Comment (mrakitin) [reply recipes/nslsii/meta.yaml]: @xhochy, are you OK with that?

Comment (mrakitin) [reply recipes/nslsii/meta.yaml]: Actually, I think these 4 lines can be added conditionally for non-Windows builds.

The same is true for `hxntools` that is failing with the same issue:
```
%SRC_DIR%>conda activate "%PREFIX%" 
import: 'hxntools'
import: 'hxntools'
nslsii 0.1.3 requires bluesky-kafka, which is not installed.
TESTS FAILED: hxntools-0.5.1-pyhd8ed1ab_0.tar.bz2
```

File: recipes/nslsii/meta.yaml [hunk 31]
```diff
@@ -0,0 +1,72 @@
...
+    - h5py
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
```
Comment (mrakitin) [inline recipes/nslsii/meta.yaml:39]:
```suggestion
test:
  requires:      # [not win]
    - pip        # [not win]
  commands:      # [not win]
    - pip check  # [not win]
```

File: recipes/hxntools/meta.yaml [hunk 32]
```diff
@@ -0,0 +1,54 @@
...
+    - ophyd
+    - pandas
+    - pyepics
+    - scipy
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (mrakitin) [inline recipes/hxntools/meta.yaml:40]:
```suggestion
  requires:      # [not win]
    - pip        # [not win]
  commands:      # [not win]
    - pip check  # [not win]
```

File: recipes/hxnfly/meta.yaml [hunk 33]
```diff
@@ -0,0 +1,45 @@
...
+    - python >=3.7
+    - hxntools
+    - ppmac
+    - xray-vision
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (mrakitin) [inline recipes/hxnfly/meta.yaml:31]:
```suggestion
  requires:      # [not win]
    - pip        # [not win]
  commands:      # [not win]
    - pip check  # [not win]
```

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes ready to merge now!

**Comment by mrakitin** [comment]
Thanks, @xhochy!

**Pull request merged**